pdf-page-count.gemspec

```diff
@@ -1,19 +1,23 @@
+$LOAD_PATH.unshift File.expand_path('../lib', __FILE__)
+name = "pdf-page-count"
+require "#{name}/version"
+
 Gem::Specification.new do |s|
-  s.name        = "pdf-page-count"
-  s.version     = "1.0.0"
+  s.name        =  name
```

Comment:
I think it'd be fine to keep "pdf-page-count" here and above in the require explicitly rather than putting it in a variable.